apis/core/test/domain/table/delete.test.ts

```diff
@@ -0,0 +1,145 @@
+import { describe, it, beforeEach } from 'mocha'
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+import clownface, { GraphPointer } from 'clownface'
+import $rdf from 'rdf-ext'
+import DatasetExt from 'rdf-ext/lib/Dataset'
+import { csvw, hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { TestResourceStore } from '../../support/TestResourceStore'
+import { NamedNode } from 'rdf-js'
+import type * as DimensionMetadataQueries from '../../../lib/domain/queries/dimension-metadata'
+import type * as TableQueries from '../../../lib/domain/queries/table'
+import type * as ColumnMappingQueries from '../../../lib/domain/queries/column-mapping'
+import '../../../lib/domain'
+import { deleteTable } from '../../../lib/domain/table/delete'
+import { ColumnMapping, Table } from '@cube-creator/model'
+
+describe('domain/table/delete', () => {
```

Comment:
Is this test suite missing the cases when deleted table removes (or not) the dimensions?

Reply:
You have the check that the dimension is deleted at line 143, I add the reverse check